### ISSUE TITLE
Automatic tests for non-x86 compile

### ISSUE THREAD

**Issue by nia-e** [rafaelcaricio/lvgl-rs#73]
Should we set up runners for other architectures besides `x86_64-unknown-linux-gnu`? If so, which? I'd suggest all current [`rustc` tier 1 targets](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-1-with-host-tools) and maybe some of the more likely-to-be-used tier 2 targets (e.g. `aarch64-unknown-linux-musl`, `armv7-unknown-linux-gnueabihf`, `arm-unknown-linux-gnueabi`, *maybe* `riscv64gc-unknown-linux-gnu`).

Also, I am completely useless at anything relating to CI and especially GitHub actions so help would be appreciated if we go ahead with this 😅 but I'l give it a shot when I can otherwise

**Comment by rafaelcaricio** [comment]
Yeah, I think this crate is more realistically going to be used in those tier 2 targets. I am not sure if there are runners for those. Probably we would need to run the tests in QEMU? 🤔
